### DISCUSSION TITLE
Allow for custom emoji / graphic symbol or no emoji for discussion categories

### DISCUSSION THREAD

**Discussion by cboelling** [community/community#3208] [Discussions] [answered]
Allow for uploading custom emoji / graphic symbol or for choosing no emoji for discussion categories in the editing categories dialogue.

**Comment by PranamBhat** [comment, marked as answer]
Hello!

You can use:

Windows logo key + . (period). The emoji keyboard will appear.

Check out more here : https://support.microsoft.com/en-us/windows/windows-10-keyboard-tips-and-tricks-588e0b72-0fff-6d3f-aeee-6e5116097942
